### ISSUE TITLE
[Bug]:X-label is not center aligned with vertical bar. VerticalBarChart

### ISSUE THREAD

**Issue by jiangdongxiaobaiban** [microsoft/fluentui#23802]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (8) x64 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
    Memory: 3.51 GB / 15.71 GB
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (103.0.1264.44)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://fluentuipr.z22.web.core.windows.net/heads/master/react-charting/demo/index.html#/examples/verticalbarchart

## Actual Behavior
The X-label is not center aligned with vertical bar, When there is only one vertical bar.

And when only two vertical in the vertical bar, they alse exist some bias.

Three and more are perfect.

## Expected Behavior
The only one vertical should be at center of x-axis.
![image](https://user-images.githubusercontent.com/29893437/177291280-549309bc-a685-4757-99a1-8a64c5e9c6d3.png)

Two vertical bar have some bias
![image](https://user-images.githubusercontent.com/29893437/177293441-ede2149e-4fed-4827-b427-5d195f213492.png)

Three vertical bar is perfect
![image](https://user-images.githubusercontent.com/29893437/177293802-c2f02f8c-c546-45c5-8d8d-2c6817072156.png)

## link to test
https://codesandbox.io/s/c0bzq?file=/src/App.tsx

### Requested priority

High

### Products/sites affected

M365 admin center

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by gouttierre** [comment]
@jiangdongxiaobaiban - To set expectations the developers will review this issue once capacity allows.

@Raghurk - Would you be able to review this issue? Thanks

**Comment by AtishayMsft** [comment]
This issue has been fixed in version 5.9.36 of the library. Could you update the version at your end
